lochness/__init__.py

```diff
@@ -305,6 +305,8 @@ def _parse_redcap(value, default_id=None):
 def _parse_mindlamp(value, default_id=None):
     '''helper function to parse a mindlamp metadata value'''
     default = 'mindlamp.*:{ID}'.format(ID=default_id)
+    if value == 'nan':
```

Comment:
Are we sure about this comparison? Shouldn't it be `if value is numpy.nan`?

Reply:
Good point. Before submitting it, I've also tested it with `math.isnan`, which didn't work. The `value` was string. I suspect the `nan` value has also been changed to string in line 147 in `lochness/__init__.py`.